fem/nonlininteg.hpp

```diff
@@ -341,6 +354,49 @@ class VectorConvectionNLFIntegrator : public NonlinearFormIntegrator
    virtual void AddMultPA(const Vector &x, Vector &y) const;
 };
 
+/** @brief This class is used to assemble the skew-symmetric form of the
```

Comment:
The comment here appears to be copy and paste from the `SkewSymmetricVectorConvectionNLFIntegrator` class. I imagine skew-symmetric form here should be convective from.

Reply:
Fixed this to correctly state it is assembling the convective form.